web/src/components/dashboard/TwoTabDashboardLayout.tsx

```diff
@@ -0,0 +1,105 @@
+import { getMergedConfig } from "@/contexts/configContext";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { filterCertifications } from "@/lib/domain-logic/filterCertifications";
+import { filterFundings } from "@/lib/domain-logic/filterFundings";
+import { Certification, Funding } from "@/lib/types/types";
+import { getVisibleCertifications, getVisibleFundings, templateEval } from "@/lib/utils/helpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+import { TabContext, TabList, TabPanel } from "@mui/lab/";
+import Tab from "@mui/material/Tab";
+import * as React from "react";
+import { ReactElement } from "react";
+import { SingleColumnContainer } from "../njwds/SingleColumnContainer";
+
+interface Props {
+  firstTab: ReactElement;
+  secondTab: ReactElement;
+  certifications: Certification[];
+  fundings: Funding[];
+}
+
+export default function TwoTabDashboardLayout({ firstTab, secondTab, certifications, fundings }: Props) {
+  const { userData } = useUserData();
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const Config = getMergedConfig();
```

Comment:
inside of components, we want to use the `useConfig` hook instead. The `getMergedConfig` helper is just for tests or helpers that can't use a hook because they're not a component